packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -74,12 +77,12 @@ export default function VisualEditor( { styles } ) {
 	const isToBeIframed =
 		( ( hasV3BlocksOnly || ( isGutenbergPlugin && isBlockBasedTheme ) ) &&
 			! hasMetaBoxes ) ||
-		renderingMode === 'template-only';
+		isEditingTemplate;
 
 	return (
 		<div
 			className={ classnames( 'edit-post-visual-editor', {
-				'is-template-mode': renderingMode === 'template-only',
+				'is-template-mode': isEditingTemplate,
```

Comment:
I couldn't find usage of this classname anywhere in the codebase.

Reply:
Hmm, true, have removed for now then.